modules/cluster-logging-rn-5.7.1.adoc

```diff
@@ -0,0 +1,27 @@
+
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-7-1_{context}"]
+= Logging 5.7.1
+This release includes: link:https://access.redhat.com/errata/RHBA-2023:3197[OpenShift Logging Bug Fix Release 5.7.1].
+
+[id="openshift-logging-5-7-1-bug-fixes"]
+== Bug fixes
+* Before this update, the presence of numerous noisy messages within Cluster Logging Operator pod logs caused reduced log readability and increased difficulty in identifying important system events. With this update, the issue is resolved by significantly reducing the noisy messages within Cluster Logging Operator pod logs, resulting in improved log readability and enhanced visibility of critical system events. (link:https://issues.redhat.com/browse/LOG-3482[LOG-3482])
+
+* Before this update, the API server would default the new collector type field to "vector" even when the custom resource had the prior field selected. This update removes the default for the new collector type field to restore the previous behavior. (link:https://issues.redhat.com/browse/LOG-4086[LOG-4086])
```

Comment:
```suggestion
* Before this update, the API server would reset the value for the `CollectorSpec.Type` field to `vector`, even when the custom resource used a different value. This update removes the default for the `CollectorSpec.Type` field to restore the previous behavior. (link:https://issues.redhat.com/browse/LOG-4086[LOG-4086])
```

Purely a suggestion, but if there's a literal name here that can be used for the field to clarify what is being referred to, I'd go with that instead of "new collector type field".

Also, put literal values in `backticks` instead of "quotation marks" per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference